### ISSUE TITLE
Hashkeys

### ISSUE THREAD

**Issue by throwawaymyrights** [Wolfsblvt/BlossomsPokemonGoManager#581]
Can somebody please explain, in layman terms, how I use a hashkey to login?

I've purchased one, but I don't really know what to do now. 
I edited the .json file and added my key. 

What now? Do I try to log in the way I normally would before I had to buy the key? Because I still get the same error as before when I try that.

**Comment by vuice** [comment]
Yes, use the app the same way as you have before.  Make sure you entered your key in the proper spot without spaces, like shown at the bottom of the following page:

https://github.com/Wolfsblvt/BlossomsPokemonGoManager

**Comment by LeonCheong** [comment]
How and where do you buy the key?

**Comment by FernandoTBarros** [comment]
talk.pogodev.org/d/54-getting-started-with-our-api-hashing-service

**Comment by FernandoTBarros** [comment]
@throwawaymyrights You had to download the 0.1.7-hotfix4 from discord channel in order to login using your hashkey. 
But we are preparing an official release here in GitHub. I suggest you to wait.

**Comment by migueltomas** [comment]
Hey guys,

Just to clarify. When are you thinking to release v1.8?

I'm not familiarize with haskeys, but if I buy it, I only have 30 days to use it? After that, I need to buy a new one?

Thanks in advance :)

**Comment by FernandoTBarros** [comment]
It's supposed to be released @migueltomas . But if you go to Discord you will get a usable hotfix version.
And yes, a hashkey lasts for a month. After that you will need to buy a new one.

**Comment by migueltomas** [comment]
@FernandoTBarros Thanks for your answer.

About usable hot fix, are you referring to v1.7 hotfix4?

I understand now. So, if i buy 150 RPM, is enough?

**Comment by FernandoTBarros** [comment]
It's already on hotfix5. Updated to the latest API 0.61 version.
Yes, 150 RPM is more than enough for at least 5 people using together.

**Comment by migueltomas** [comment]
@FernandoTBarros Can you share discord link? Because i can't find it :/

EDIT: I found it, LOL. Sorry.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using BlossomsPokemonGoManager. This project is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving it. Please do not use it with a live account or current credentials. See the [Disclaimer](https://github.com/Wolfsblvt/BlossomsPokemonGoManager#disclaimer) for details.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. Blossoms Pokémon Go Manager is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving the repository. This avoids leaving a false impression that further development or support is planned.
